File: Tareas/24/Código JAVA/tarea24.java
/*	
	Nombre: Leslie Estefanía Pérez Cerda
	Matrícula: 1737535
	Hora: Jueves V4-V6
*/

import javax.swing.JOptionPane;
public class tarea24{
	public static void main (String[] args){
		String entrada ="";
		String salidaA = "";
		String salidaB = "";
		String salidaC = "";
		int i, j, c, f, b, suma=0;

		entrada = JOptionPane.showInputDialog(null,"Ingrese el numero de filas:");
		c = Integer.parseInt(entrada);
		entrada = JOptionPane.showInputDialog(null,"Ingrese el numero de columnas:");
		f = Integer.parseInt(entrada);
		int[][] matrizA = new int[c][f];
		int[][] matrizB = new int[c][f];
		int[][] matrizC = new int[c][f];

		for(i=0; i<c; i++){
			for(j=0; j<f; j++){
				entrada = JOptionPane.showInputDialog(null,"Ingrese el valor de matrizA["+i+"]["+j+"]");
				matrizA[i][j] = Integer.parseInt(entrada);
			}
		}
		for(i=0; i<c; i++){
			for(j=0; j<f; j++){
				entrada = JOptionPane.showInputDialog(null,"Ingrese el valor de matrizB["+i+"]["+j+"]");
				matrizB[i][j] = Integer.parseInt(entrada);
			}
		}

		for(i=0; i<c; i++){
			for(j=0; j<f; j++){
				matrizC[i][j] = matrizA[i][j] + matrizB[i][j];

				salidaA = salidaA + matrizA[i][j] + "   ";
				salidaB = salidaB + matrizB[i][j] + "   ";
				salidaC = salidaC + matrizC[i][j] + "   ";
			}
			salidaA = salidaA + ("\n");
			salidaB = salidaB + ("\n");
			salidaC = salidaC + ("\n");
		}

		JOptionPane.showMessageDialog(null,"[A]\n"+salidaA+"+\n[B]\n"+salidaB+"\n=\n[C]\n"+salidaC);
	} //Fin de función main
} //Fin de clase tarea24

File: Tareas/24/Código RUBY/tarea24.rb
=begin
	Nombre: Leslie Estefanía Pérez Cerda
	Matrícula: 1737535
	Hora: Jueves V4-V6
=end

salidaC = ""

print("\nIngrese el numero de filas de [A] --> ")
fA = gets().to_i
print("Ingrese el numero de columnas de [A]--> ")
cA = gets().to_i
matrizA = Array.new(fA){Array.new(cA)}

for i in (0...fA)
   for j in (0...cA)
      print("Ingrese el valor de A[#{i}][#{j}]: ")
	  matrizA[i][j] = gets().to_i
   end
end

print("\nIngrese el numero de filas de [B] --> ")
fB = gets().to_i
print("Ingrese el numero de columnas de [B]--> ")
cB = gets().to_i
matrizB = Array.new(fB){Array.new(cB)}

for i in (0...fB)
   for j in (0...cB)
      print("Ingrese el valor de B[#{i}][#{j}]: ")
	  matrizB[i][j] = gets().to_i
   end
end

if(cA == fB)
	matrizC = Array.new(fB){Array.new(cA)}

	for i in(0..fB)
		for j in(0..cA)
			matrizC[i][j] = matrizC[i][j] + matrizA[i][j] + matrizB[j][i]
		end
	end

	for i in (0...fB)
   		for j in (0...cA)
      		salidaC = salidaC + matrizC[i][j].to_s + " "
   		end
   		salidaC = salidaC + "\n"
	end

elsif(cB == fA)
	matrizC = Array.new(fA){Array.new(cB)}

	for i in(0..fA)
		for j in(0..cB)
			matrizC[i][j] = matrizC[i][j] + matrizB[i][j] + matrizA[j][i]
		end
	end

	for i in (0...fA)
   		for j in (0...cB)
      		salidaC = salidaC + matrizC[i][j].to_s + " "
   		end
   		salidaC = salidaC + "\n"
	end

else
	print("\nLas matrices no son multiplicables")
end

print("\n[A] * [B] = [C]\n\n#{salidaC}")